deploy/ocp-templates/glusters3template.json

```diff
@@ -0,0 +1,205 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "glusters3template",
+        "creationTimestamp": null,
+        "annotations": {
+          "description": "Gluster s3 service template",
+          "tags": "glusterfs,heketi"
+        }
+    },
+    "objects": [
+        {
+            "kind": "Pod",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "glusters3",
+                "creationTimestamp": null,
+                "labels": {
+                    "name": "glusters3"
+                },
+                "annotations": {
+                    "openshift.io/scc": "privileged"
+                }
+            },
+            "spec": {
+                "volumes": [
+                    {
+                        "name": "glusterfs-cgroup",
+                        "hostPath": {
+                            "path": "/sys/fs/cgroup"
+                        }
+                    },
+                    {
+                        "name": "gluster-vol1",
+                        "persistentVolumeClaim": {
+                            "claimName": "glusterfs-claim"
+                        }
+                    },
+                    {
+                        "name": "gluster-vol2",
+                        "persistentVolumeClaim": {
+                            "claimName": "glusterfs-claim-meta"
+                        }
+                    }
+                ],
+                "containers": [
+                    {
+                        "name": "glusters3",
+                        "image": "brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/rhgs3/rhgs-s3-server-rhel7:3.3.0-2",
```

Comment:
The registry path should not be hardcoded? Won't  rhgs3/rhgs-s3-server-rhel7:3.3.0-2 suffice?